src/lib/analytics.js

```diff
@@ -213,6 +245,8 @@ async function initAnalytics() {
   onINP(sendToGoogleAnalytics);
   onLCP(sendToGoogleAnalytics);
   onTTFB(sendToGoogleAnalytics);
+
+  logPrerenders();
```

Comment:
You previously had some code here to feature-detect `speculationrules` support. I think you want to add that back, no? (Either here or inside the `logPrerenders()` function).

Reply:
Added back in.